src/main/java/address/controller/RootLayoutController.java

```diff
@@ -82,7 +75,7 @@ private FileChooser getXmlFileChooser() {
     @FXML
     private void handleNew() {
         logger.debug("Wiping current model data.");
-        PrefsManager.getInstance().clearSaveLocation();
+        modelManager.getPrefs().clearSaveLocation();
```

Comment:
Will do. How about for components that takes in `UserPrefs` in constructor? Should I make an interface that allows them to read only?